mm-go-irckit/userbridge.go

```diff
@@ -138,22 +140,53 @@ func (u *User) handleDirectMessageEvent(event *bridge.DirectMessageEvent) {
 	}
 	text, prefix, suffix, showContext, maxlen := u.handleMessageThreadContext(prefixUser, event.MessageID, event.ParentID, event.Event, event.Text)
 
+	syntaxHighlighting := u.v.GetString(u.br.Protocol() + ".syntaxhighlighting")
+	formatter := "terminal256"
+	style := "pygments"
```

Comment:
hm this enables it by default for everyone, should be empty strings right?
(same remark on 325,326)

Reply:
Ah yes, fixing that. Also found a bug with `PrefixContext = true` + `ShowContextMulti = true`. Working on fixing both.

Reply:
Oh, it doesn't matter. That sets the default, it doesn't enable for everyone though because:

`else if (codeBlockBackTick || codeBlockTilde) && syntaxHighlighting != "" && !startSyntaxHighlight && lexer != "" {`

So `syntaxHighlighting` needs to be non-empty.